.github/trigger_script.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+# Check all pyproject.toml files that have changed
+for file in $(git diff --name-only HEAD~1..HEAD | grep pyproject.toml); do
+    # Extract the current version and buikd the expected tag
+    dirpath=$(dirname $file)
+    expected_tag=$(.github/helper.sh $dirpath --path-version)
+    version=$(.github/helper.sh $dirpath)
+    # Check if the tag already exists
```

Comment:
Ideally, here we would check if the tag is a valid version bump to the existing one. It would be a nice to have, to ensure that we don't accidentally bump v1.1.0 to v1.5.0 for instance.
However, in bash this would be tricky to achieve. I think that in the Go SDK we have a Go script to handle this.
At the end of the day, its not a big deal, so we can keep it like this if you think its the best decision

Reply:
We had the discussion if we want to validate if the version is correct but decided against it.